Makefile

```diff
@@ -265,9 +265,12 @@ update_devworkspace_crds:
 	if git show-ref --verify refs/tags/$(DEVWORKSPACE_API_VERSION) --quiet; then
 		echo 'DevWorkspace API is specified from tag'
 		git checkout tags/$(DEVWORKSPACE_API_VERSION)
-	else
+	elif git rev-parse --verify $(DEVWORKSPACE_API_VERSION); then
```

Comment:
Now I'm not sure it works fine...Maybe it's better to remove it for time being.

Reply:
It at least doesn't interfere with release, so we can always fix the makefile later (as long as we have the right CRDs used for generation)